independent-projects/bootstrap/runner/src/main/java/io/quarkus/bootstrap/runner/JarResource.java

```diff
@@ -185,6 +185,8 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(manifestInfo, jarPath);
+        int result = Objects.hashCode(manifestInfo);
+        result = 31 * result + Objects.hashCode(jarPath);
+        return result;
```

Comment:
Ah yeah, we banned `Objects.hash` from Hibernate Validator for this reason.

I think we can even go further and remove the `manifestInfo` from the `equals()` and `hashCode()`: the unicity should be done on the `Path` really, we don't care about what's in the `Manifest`, we will never have a jar with the same `Path` and a different `ManifestInfo`.

@aloubyansky could you confirm ^?

Reply:
Right, sounds good. Thanks!

Reply:
@franz1981 can you adjust or do you want me to take care of it?

Reply:
If you could (and put you as co-author if you wish) would be great...I'm still catching up with the email :(

Reply:
Sure will do, didn't want to step on your toes :). Will add a separate commit as I think we need to keep the semantic of both changes.